scripts/scr_ui_formation_bars/scr_ui_formation_bars.gml

```diff
@@ -1,241 +1,107 @@
 // Creates the formation bars and draws them according to their designated type (infantry and vehicles)
 function scr_ui_formation_bars() {
-	var x9,y9,bar=0,ii=0,nbar=0,abar=0,fo=formating,te=4700;
+    var x9, y9, bar = 0,
+        ii = 0,
+        nbar = 0,
+        abar = 0,
+        fo = formating,
+        te = 4700;
 
-	x9=__view_get( e__VW.XView, 0 )+49;
-	y9=__view_get( e__VW.YView, 0 )+224;
+    x9 = __view_get(e__VW.XView, 0) + 49;
+    y9 = __view_get(e__VW.YView, 0) + 224;
 
-	with(obj_formation_bar){instance_destroy();}
-	with(obj_temp8){instance_destroy();}
-	
-	for (bar = 1; bar <=10; bar++){
-		te+=1;
-		ii=0;
-		temp[te]=0;
-	    var cu=instance_create(x9,y9,obj_temp8);
-		cu.col_parent=bar;
-    
-	    temp[te]=0;
-		temp[te+100]=0;
-    
-		for(ii = 1; ii<=15; ii++){
```

Comment:
this for loop is not needed, we can just init combat bars in the for(bar = 1;...) loop directly